### PR TITLE
Adds a marine attachment that makes your bullets faster

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Makes your bullets faster, thats it

## Why It's Good For The Game
It makes bullets faster at the cost of not having any other barrel attachment

## Changelog
:cl:
balance: added a marine barrel attachment that makes bullets faster
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
*But MJP, Extended Barrel already does that!*

**Comment by Pariah919** [comment]
>But MJP, Extended Barrel already does that!

FASTER

**Comment by jroinc** [comment]
> 
> 
> > But MJP, Extended Barrel already does that!
> 
> FASTER

Powercreep go BRRRRR

**Comment by David-Stormwell** [comment]
bbbbbbbbbb uuuuuu tttttt exxxxxtttteeenndddd baarrrrweewwwelll

**Comment by lovegreenstuff** [comment]
barrel charger.

that is all

**Comment by Losenis** [comment]
MAKE BULLETS HITSCAN

**Review by TiviPlus — CHANGES_REQUESTED** [review]
A) give it a downside you noob
B) Extended barrel exists

**Comment by SplinterGP** [comment]
> A) give it a downside you noob
> B) Extended barrel exists

Its downside is not using other attachments, kinda, still though it sucks

**Comment by David-Stormwell** [comment]
> A) give it a downside you noob
> B) Extended barrel exists

To be honest, not having another barrel attachment is a pretty good downside as ext and rc are already great.
And faster bullets are to be honest... Interesting.
I'd like to see this

**Comment by TiviPlus** [comment]
Not having a downside isn't a downside

**Comment by Pariah919** [comment]
it has minor downsides

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -410,14 +410,13 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
...
 	slot = ATTACHMENT_SLOT_MUZZLE
 	icon_state = "hbarrel"
 	attach_icon = "hbarrel_a"
-	accuracy_mod = -0.6
-	damage_mod = 0.1
-	scatter_mod = 25
-	accuracy_unwielded_mod = -0.3
+	attach_shell_speed_mod = 2
+	accuracy_mod = -0.1
+	scatter_mod = 2
```
Comment (Pariah919) [inline code/modules/projectiles/gun_attachables.dm:419]:
```suggestion
```

**Comment by SplinterGP** [comment]
> 
> 
> Not having a downside isn't a downside

the other barrels dont have downsides, their "Downside" is theres another option which has different stats that are better in X

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**